src/resolvers-cassandra/Messages/queries.js

```diff
@@ -12,26 +12,33 @@ const trackEvent = require('../../clients/appinsights/AppInsightsClient').trackE
  */
 
 function eventToFeature(row) {
+  const FeatureType = 'MultiPoint';
```

Comment:
General question: what's the point of having this if there's only one acceptable value? The schema defines this like so:

```
  enum FeatureType {
    MultiPoint
  }
```